.buildkite/pull-requests.json

```diff
@@ -9,8 +9,8 @@
             "set_commit_status": true,
             "build_on_commit": true,
             "build_on_comment": true,
-            "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
-            "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
+            "trigger_comment_regex": "^/test (.*)$",
+            "always_trigger_comment_regex": "^/test (.*)$",
```

Comment:
as discussed offline, if we don't make use of capture groups, there's no need to use parens.